build/test.js

```diff
@@ -1,7 +1,9 @@
+'use strict';
+
 const Builder = require('./Builder.js');
 
-let build = new Builder();
-console.log("Created builder :)");
+const build = new Builder();
+console.log('Created builder :)');
 
 //do BYBA colours
-build.setVariable("default", "#282360").build(__dirname + "/../scss/style.scss", __dirname + "/../style.css");
+build.setVariable('default', '#282360').build(__dirname + '/../scss/style.scss', __dirname + '/../style.css');
```

Comment:
???

Reply:
There is a rule to standardise quote styles and I sort of abitrarily chose `'` over `"` because that's what the linting package was written in - if we want to change this then it's dead easy.

Reply:
I feel like this rule might become more frustrating than useful tbh... where does it stand on backticks?

Reply:
It allows backticks as long as they're necessary, eg if there is a template literal in there. But if not it will recommend that you use a string instead.

The rule can be removed if needed - it was just sort of a skeleton of some rules I use; I'm sure we'll find ones we don't like and weed them out.

Reply:
I'd say take this rule out for now; both `'` and `"` are equally valid quotes so limiting ourselves to one or the other doesn't seem optimal imo.

Fair enough if it was e.g. C# where one denotes a `string` and the other a `char`, but I don't see the benefit here

If it causes problems further down the line then we can absolutely look at pulling this back in

Reply:
Agreed - as above I'll look into simplifying the rules down to what we really need then